### ISSUE TITLE
Exception Occured. Only on iPhone, iPad ist running without problems.

### ISSUE THREAD

**Issue by wir43** [lwitzani/homebridgeStatusWidget#23]
Hey, got an issue on my iPhone 12 Pro with iOS 14.2.1. On my iPad the same Scriptabel script is running without any error.
When I start the Script on my iPhone I got the following error:

Exception Occured
Error on line 582:22: TypeError: null is not an object (evaluating ´CONFIGURATION.hbServiceMachine-BaseURL`)

The version of the Script is from 2 days ago, so the actual version. The scripts in the Scriptable App are synced over iCloud, so there is the same version on both devices. Can`t figure out, where the problem is.

**Comment by lwitzani** [comment]
Never saw that kind of error before. It seems that the CONFIGURATION is null at that line. Can you rename the config.json on the iphone so it does not use the exact same as on the ipad?

**Comment by wir43** [comment]
Thanks, tested it an it seems to be a problem, when the same config file is used on both devices.
On the device, the config file is created, the script is running. On the 2nd device, the error occured.

I renamed the config file on the iPhone, the script runs without an error.
The same script on the iPad showed the old error.

Because of the sync in the scriptable app between both devices, I created two scripts. One für the iPad and a copy with an other config name for the iPhone.

So for the moment the problem is solved for me with this workaround, thanks.
